packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -674,7 +689,10 @@ function getParameterValue<T extends NodeParameterValueType = NodeParameterValue
 	.parameter-item {
 		position: relative;
 		margin: var(--spacing-xs) 0;
+		// width: calc(100% - 28px);
 	}
+	.parameter-item:hover > .drag-option,
+	.multi-parameter:hover > .drag-option,
 	.parameter-item:hover > .delete-option,
 	.multi-parameter:hover > .delete-option {
```

Comment:
I think we can reduce the duplication in the styles a bit by using 1 class for both these buttons (`.action` ?)

Reply:
I changed it to icon-button, the way FixedCollectionParameter and this component are unnecessary coupled requires some refactoring but maybe can be done in another PR